connector/manifest.xml

```diff
@@ -1,14 +1,14 @@
 <?xml version='1.0' encoding='utf-8' ?>
 
-<connector-plugin class='mongosql_jdbc' superclass='jdbc' plugin-version='0.0.0' name='MongoSQL' version='18.1' min-version-tableau='2020.4'>
+<connector-plugin class='mongosql_jdbc' superclass='jdbc' plugin-version='0.1.0' name='MongoSQL' version='18.1' min-version-tableau='2021.1'>
   <vendor-information>
       <company name="MongoDB"/>
-      <support-link url="https://example.com"/>
-      <driver-download-link url="https://drivers.example.com"/>
+      <support-link url="https://support.mongodb.com"/>
+      <driver-download-link url="https://search.maven.org/search?q=a:mongodb-jdbc"/>
```

Comment:
```suggestion
      <driver-download-link url="https://search.maven.org/artifact/org.mongodb/mongodb-jdbc"/>
```

Reply:
Another thought/question: where does this download link appear in the connector? Is it intended for users to follow and download manually, or is it used automatically by Tableau somehow?

Reply:
I see this download link show up in the error dialogue when the jdbc driver isn't present in the Tableau/Drivers/ directory.  It's meant for the user to follow and install manually.  With the following text:
```
No suitable driver installed, or the URL is incorrect
Connector Class: mongosql_jdbc, Version: 0.1.0
[Download and install the drivers, and then connect.](https://search.maven.org/artifact/org.mongodb/mongodb-jdbc)
[For support, contact MongoDB.](https://support.mongodb.com/)
```